app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class QuestionsController < ApplicationController
+
+  def index
+    @questions = Question.all
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @new_question = Question.new(question_params)
+    if @new_question.save!
+      redirect_to question_path(@new_question)
+    else
+      flash[:notice] = "A problem has occurred. Your question couldn't be posted."
+      redirect_to questions_path
+    end
+  end
+
+  def show
+    @question = Question.find(params[:id])
+    @question_vote = @question.votes
+
+  end
+
+  def destroy
+    @question = Question.find(params[:id])
+    if session[:user_id] == @question.creator_id

```

Comment:
It might be a good idea to encapsulate who can edit / destroy questions / answers on the model.